convert2rhel/actions/report.py

```diff
@@ -16,14 +16,33 @@
 __metaclass__ = type
 
 import logging
+import textwrap
 
-from convert2rhel.actions import find_actions_of_severity, format_report_message
+from convert2rhel import utils
+from convert2rhel.actions import find_actions_of_severity, format_report_message, format_report_section_heading
+from convert2rhel.logger import bcolors, colorize
 
 
 logger = logging.getLogger(__name__)
 
+#: Map Status codes (from convert2rhel.actions.STATUS_CODE) to color name (from
+#: convert2rhel.logger.bcolor)
 
-def summary(results, include_all_reports=False):
+_STATUS_TO_COLOR = {
+    # SUCCESS
+    0: "OKGREEN",
+    # WARNING
+    51: "WARNING",
+    # SKIP
+    101: "FAIL",
+    # OVERRIDABLE
+    152: "FAIL",
+    # ERROR
+    202: "FAIL",
+}
+
+
+def summary(results, with_colors=True, include_all_reports=False):
```

Comment:
Why not including `with_colors` as a latter parameter instead? It seems to me that this will be always a optional thing that will be enabled for most cases. Would simplify the usage as well if someone doesn't use named parameters to call this 

```suggestion
def summary(results include_all_reports=False, with_colors=True,):
```

Reply:
Also, thinking more about this now, wouldn't be easier to make the default value for `with_colors` the output of `should_disable_color_output`? 

```suggestion
def summary(results, include_all_reports=False, with_colors=should_disable_color_output()):
```

Reply:
Makes sense to me. We can keep the way it is.